### PR TITLE
chore(common): Merge beta to master for Sprint B17S5

### PR DESCRIPTION
**Pull request by darcywong00**
End of sprint task

Curious the 🪠emojis aren't stripped out in HISTORY.md

@keymanapp-test-bot skip

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -75,6 +75,81 @@
...
 * chore: move to 18.0 alpha
 
+## 17.0.306 beta 2024-04-11
+
+* docs(ios): updates iOS app help for 17.0 banner changes (#11200)
+* chore(oem/fv): Add fv sguuxs and update keyboard versions (#11198)
+
+## 17.0.305 beta 2024-04-10
+
+*  (#11169)
```
Comment (jahorton) [inline HISTORY.md:85]:
```suggestion
*  fix(core): skip leading trail surrogate char in km_core_state_context_set_if_needed() (#11169)
```

File: HISTORY.md [hunk 2]
```diff
@@ -75,6 +75,81 @@
...
+* change(web): merges split async method in gesture engine (#11142)
+* fix(web): blocks nextLayer for keys quickly typed when multitapping to new layer when final tap is held (#11189)
+* refactor(web): OSK spacebar-label updates now managed by layer object (#11175)
+
+## 17.0.304 beta 2024-04-09
+
+* fix(android): atomically updates selection with text (#11188)
+*  (#11178)
```
Comment (jahorton) [inline HISTORY.md:93]:
```suggestion
*  fix(web): fix crash in nearest-key row lookup when touch moves out-of-bounds (#11178)
```

File: HISTORY.md [hunk 3]
```diff
@@ -75,6 +75,81 @@
...
+* fix(web): early gesture-match abort when unable to extend existing gestures 🪠 (#10836)
+* fix(web): infinite model-match replacement looping 🪠 (#10838)
+* fix(web): proper gesture-match sequencing 🪠 (#10840)
+* change(web): input-event sequentialization 🪠 (#10843)
+* fix(web): proper linkage of sources to events 🪠 (#10960)
+* fix(developer): handle buffer boundaries in four cases (#11137)
+* chore(linux): Build packages for next Ubuntu version separately (#11153)
+* fix(common): upgrade sentry-cli to 2.31.0 (#11151)
+* fix(android/app): Track previous device orientation for SystemKeyboard (#11134)
+*  (#11129)
```
Comment (jahorton) [inline HISTORY.md:118]:
```suggestion
* change(web): reworks nearest-key detection to avoid layout reflow (#11129)
```

**Review by jahorton — APPROVED** [review]

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 18.0.18-alpha](https://keyman.com/downloads/releases/alpha/18.0.18)

**Comment by mcdurdin** [comment]
> Curious the 🪠emojis aren't stripped out in HISTORY.md

#11220

**Pull request merged**